src/internal/Confirm.tsx

```diff
@@ -0,0 +1,165 @@
+import * as React from 'react';
+import { Drop, DropProps } from 'grommet';
+import styled from 'styled-components';
+import { px } from '../utils';
+import { Box, Button } from '..';
+import TooltipArrow, { TooltipArrowProps } from './TooltipArrow';
+
+const StyledDrop = styled(Drop)`
+	border: 0;
+`;
+
+const StyledDropInner = styled.div`
+	margin: 5px;
+	background: white;
+	box-shadow: ${(props) => '1px 1px 5px' + props.theme.colors.gray.light};
+	border-radius: ${(props) => px(props.theme.radius)};
+	border: ${(props) => '1px solid ' + props.theme.colors.gray.main};
```

Comment:
Can we make the triangle 'background' to be the same color as the background of the `Drop` (with no line between them)

(In due course we'll have to replace 'white' with a theme color as well but I appreciate there isn't a suitable defined color at the moment! We're working on this!)